core/Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/ipfs-rust/rust-ipld"
 default = ["std"]
 std = ["anyhow/std", "cid/std", "multibase/std", "multihash/std", "thiserror"]
 serde-codec = ["cid/serde-codec", "serde"]
+quickcheck = ["dep:quickcheck"]
```

Comment:
I wonder if it would make sense to follow rust-cid and rust-multihash and name this feature `arb` instead. This way it could potentially also provide an implementation for https://crates.io/crates/arbitrary.

Related: would it make sense to remove the `dep:`? It would make the library need a quite new rust stable version, and I prefer bumping it only when there's a good reason to.